<?xml version="1.0"?>

<launch>
    <rosparam command="load" file="$(find putm_dv_perception)/config/config.yaml" />
    <node type="lidar_reciver.py" name="lidar_reciver" pkg="putm_dv_perception" output="screen" />

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find putm_dv_perception)/rviz/rviz_config.rviz" />
</launch> 